pkg/systemd/dbus.go

```diff
@@ -37,13 +37,26 @@ func (c *DbusConn) IsActive(ctx context.Context, unitName string) (bool, error)
 	if !c.conn.Connected() {
 		return false, fmt.Errorf("connection disconnected")
 	}
-	if !strings.HasSuffix(unitName, ".target") && !strings.HasSuffix(unitName, ".service") {
```

Comment:
IMO we should define an interface for dbus.Conn, we should add an interface to DbusConn,  and we should generate a mock for DbusConn, so that other codes rely on the DbusConn can be testable.

Reply:
done